app/controllers/application_controller.rb

```diff
@@ -1,8 +1,11 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :current_user
```

Comment:
- [ ] Kindly remove the current_user method in ApplicationController, Devise provides a method to do this 👍🏿